pkg/sti/script/installer.go

```diff
@@ -108,6 +109,20 @@ func (s *handler) download(scripts []api.Script, workingDir string) (bool, error
 		}
 	}
 
+	// check for the scripts inside the sources
+	scriptsDir := filepath.Join(workingDir, "/upload/src/.sti/bin")
+	if s.fs.Exists(scriptsDir) {
+		for _, script := range scripts {
+			file := filepath.Join(scriptsDir, string(script))
+			if s.fs.Exists(file) {
+				downloads[script] <- true
+			} else {
+				errs[script] <- errors.NewScriptDownloadError(script, nil)
```

Comment:
it's not necessarily an error if it's not there, right?
1) it could be an optional script
2) as i recall we union all the sources of scripts together, so you might get just the "run" script from the src dir, and "assemble" comes from the default scripts url.